photon-api/src/main/scala/com/linkedin/photon/ml/algorithm/CoordinateFactory.scala

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Missing unit tests for `CoordinateFactory`.

Reply:
Added